common/configure

```diff
@@ -161,7 +161,8 @@ printf "DEST=\$(DESTDIR)\$(PREFIX)\n\n" >> ${MAKEFILE}
 
 printf "all:\tbuild\n\n" >> ${MAKEFILE}
 
-printf "build:\ttranslate compress\n\n" >> ${MAKEFILE}
+printf "build:\ttranslate compress\n" >> ${MAKEFILE}
+printf "\t(cd .. && ./updateversion.sh)\n\n" >> ${MAKEFILE}
```

Comment:
I think changing the directory within the script may have side effects on all other relative paths in commands called subsequently (= using the wrong path).

AFAIR a solution for this would be to `cd` in the called `updatevesion.sh` script (into the folder of the script).
I need to search for a `bash` code for this (can't find it ATM)

Reply:
It is IMHO no problem because this "cd" is temporary because of the `(...)`. It is activte only inside the brackets.

```
$ ls && (cd .. && ls) && ls
huhu.txt
downdir/  infoobar.txt
huhu.txt
```

It seems to be POSIX and works on several shells. The make system do use "sh" by default. There is also the `pushd` command but it works only on `bash`. You would do `ls && pushd .. && ls && pushd +1 && ls` on bash.

Reply:
Magic - looks very good (didn't know the parentesis semantics until now :-)